### PR TITLE
Added function to access whether royal pontoon.

### PR DESCRIPTION
**Pull request by notsoprocoder**
Function takes a hand in the format [{card-object-1}, {card-object-2}] and determines whether it contains a royal face and an ace.

### CODE REVIEW

**Comment by dhuber666** [comment]
Hi @notsoprocoder 
First of all welcome to the world of github and nice that you took the first step 👍 
**To your code:**

I like it and it's actually a good  function. One thing you have to consider though is in which object it fits into. My code is written into "objects" or you can think of it as "classes". 
These are the functions that starts with Uppercase letter. I don't know how familiar you are with it. 
You could for example add this function as a prototype function to the Dealer or even better maybe create a new constructor Function (Object) that acts like the "game loop" and maybe calculates all the stuff that's going on. 
Also make sure to fetch new changes I added a few new things last couple minutes. 

If you change it to fit into the existing code (small change) I will merge it. 
**Keep up the good work!**

**Comment by notsoprocoder** [comment]
Thanks for the guidance - I added the function to an object but I kept getting conflict errors because I didn't merge the branch. I have run out of time this morning. I will look into it this PM. thanks for your help.

**Comment by notsoprocoder** [comment]
I think I managed to commit the new updates. Let me know, if it is alright.

**Comment by dhuber666** [comment]
So one more thing before I will merge your commit.
I like the function and it will get pretty useful soon (in the simulate method). 

One thing I want you to change is that it should be a prototype function of the Table object instead of the Player. Maybe later on I will add a Util Object where such "helper" method will live.

File: tests.js [hunk 1]
```diff
@@ -198,3 +201,16 @@ describe('Table', function() {
 		});
 	});
 });
+
+describe('twoCardHand', function(){
```
[the remark this answers is not in the document]
Comment (notsoprocoder) [reply tests.js]: I thought I'd removed that peice of the code, I was playing around with Jasmine.

**Comment by notsoprocoder** [comment]
Hey,
I've updated to removed the test and moved the function into the Table object. Do I need to submit a new pull request?
Thanks for all your help!

**Comment by dhuber666** [comment]
No I don't see the changes yet. I thing you have to push the changes to your forked master or make a new commit with the changes. Not 100 % sure.

**Comment by notsoprocoder** [comment]
I appreciate you're patience - I'm trying to write a test for pontoon and 5 card trick but I cannot figure out how to call the functions in the test file. Could you give me an example?

**Comment by dhuber666** [comment]
Thanks for the contribution and grats for your first successful pull request :) 
Please look at the readme file again. Basic functionality of the app is now up and running 👍

**Comment by dhuber666** [comment]
@notsoprocoder to your other question.

jasmin has a describe block. Into this describe block you nest your tests with it and give it a function.
After you describe a block you can run a beforeEach() block which runs before each test. So the way I set this up is that it instantiates a new player / deck, table what ever before each test, so I don't have to do this on every file. 

So for example if you have this test 
```
describe('Simulate', function() {
		var players;
		var deck;
		var dealer;
		var table;

		beforeEach(function() {
			players = [new Player('dominik'), new Player('magdalena')];
			deck = new Deck();
			dealer = new Dealer(deck);
			table = new Table(dealer, players);
		});

		it('should give 2 cards to every player in the players array when simulation runs 1 round', function() {
			table.simulate(1);

			expect(table.players[0].hand[0]).toBeTruthy();
			expect(table.players[1].hand[1]).toBeTruthy();
			// 3rd card should be false (just 2 cards)
		});
```

You could run a function sitting on the table like this:
`table.anyFunctionOnTheTable`.
If you want to run a function on the dealer that "sits" on the table you call it like this:
`table.dealer.anyFunctionOnTheDealer`

**Pull request merged**